README.md

```diff
@@ -141,9 +144,7 @@ If you like this project, you are most welcome to Contribute.
 
```

Comment:
- [ ] It would be nice if you fix your linters because your linters are not running 😨 Kindly fix your conflict because this might be an issue of your linters breaking 👍